### PR TITLE
fix(apps/interface): fix workflow failed to start

### PR DESCRIPTION
**Pull request by asaadam**
fix failed to start

### CODE REVIEW

File: .github/workflows/risedle-interface-stats.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
           node-version: "17.8.0"
 
       - name: Install dependencies
-        uses: npm install
+        run: npm install
```
Comment (pyk) [inline .github/workflows/risedle-interface-stats.yml:30]: I think we need to skip the husky install on CI

```
> monorepo@0.0.0 prepare
> husky install

/tmp/prepare-53a542ad.sh: 1: husky: not found
npm ERR! code 127
npm ERR! path /home/runner/work/monorepo/monorepo
npm ERR! command failed
npm ERR! command sh -c npm run prepare
```

**Review by pyk — CHANGES_REQUESTED** [review]

**Comment by pyk** [comment]
@asaadam it seems better to remove all `cd ../..` and add this to analysis step:

```
working-directory: ./apps/interface
```

**Comment by asaadam** [comment]
> @asaadam it seems better to remove all `cd ../..` and add this to analysis step:
> 
> ```
> working-directory: ./apps/interface
> ```

oaky ser

**Review by pyk — CHANGES_REQUESTED** [review]

File: .github/workflows/risedle-interface-stats.yml [hunk 2]
```diff
@@ -41,25 +40,26 @@ jobs:
...
         with:
           name: bundle
-          path: .next/analyze/__bundle_analysis.json
+          path: ./apps/interface/.next/analyze/__bundle_analysis.json
 
       - name: Download base branch bundle stats
         uses: dawidd6/action-download-artifact@v2
         if: success() && github.event.number
         with:
+          working-directory: ./apps/interface
```
[the remark this answers is not in the document]
Comment (asaadam) [reply .github/workflows/risedle-interface-stats.yml]: updated ser

**Review by pyk — APPROVED** [review]
Ok let's try it

**Pull request merged**